x-pack/platform/plugins/shared/cases/public/components/case_view/components/case_view_events.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexItem } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { type CaseUI } from '../../../../common';
+import { CASE_VIEW_PAGE_TABS } from '../../../../common/types';
+import { CaseViewTabs } from '../case_view_tabs';
+import type { CaseViewProps } from '../types';
+
+import { AttachmentType, type EventAttachment } from '../../../../common/types/domain';
+
+interface CaseViewEventsProps {
+  caseData: CaseUI;
+  renderEventsTable: CaseViewProps['renderEventsTable'];
+}
+
+export const CaseViewEvents = ({
+  caseData,
+  renderEventsTable: EventsTable,
+}: CaseViewEventsProps) => {
+  const events = useMemo(
+    () =>
+      caseData.comments
```

Comment:
💅 In the future we might want to move the event and alert filtering to a backend endpoint so we do not need to filter these on the frontend